render/meson.build

```diff
@@ -1,13 +1,33 @@
+renderers = get_option('renderers')
+egl = dependency('egl', required : false)
```

Comment:
EGL is required for the GLES2 renderer.

Reply:
For the current code, it is, in theory  no. There is opengl offscreen, that would be used for those headless backend